cadence/contracts/templates/emulator/EVMBridgedNFTTemplate.cdc

```diff
@@ -28,7 +28,7 @@ import CrossVMNFT from 0xf8d6e0586b0a20c7
 /// bridging methods which will programatically route bridging calls to this contract.
 ///
 // TODO: Implement NFT contract interface once v2 available locally
-access(all) contract CONTRACT_NAME : ICrossVM, IEVMBridgeNFTMinter, ViewResolver {
+access(all) contract CONTRACT_NAME : ICrossVM, IEVMBridgeNFTMinter, NonFungibleToken {
```

Comment:
Is there a better way we can template 'CONTRACT_NAME'? i.e. {CONTRACT_NAME} or something

Reply:
Yeah, that works as well. The curly braces matches other templating patterns so it's more apparent that the `CONTRACT_NAME` value is a placeholder

Reply:
Updated to `{{CONTRACT_NAME}}`, borrowing from [moustache](https://mustache.github.io/)